src/onnx/parse_resize.cpp

```diff
@@ -33,49 +33,28 @@ namespace migraphx {
 inline namespace MIGRAPHX_INLINE_NS {
 namespace onnx {
 
-static std::vector<int>
-calc_neighbor_points(const std::vector<std::vector<std::vector<std::size_t>>>& vvv_ind,
-                     int i_dim,
-                     std::vector<std::vector<std::size_t>> vec_dims,
-                     const shape& in_s)
+static void calc_neighbor_points(const std::vector<std::vector<std::vector<std::size_t>>>& vvv_ind,
+                                 const std::size_t& n_dims,
+                                 const std::size_t& out_elements,
+                                 const shape& in_s,
+                                 std::vector<int>& vec_ind)
 {
-    if(i_dim == vvv_ind.size())
+    for(std::size_t start = 0; start < (std::size_t{1} << n_dims); start++)
```

Comment:
`start < n_dims * 2`. I don't know the logic here, but the loop can be simplified a bit.

Reply:
You're relying on bitshifts I suppose to save cycles then? 1 cycle vs the more for mul/divides right?

Reply:
That's right. And semantically the loop is to take each bit  so shift left to get the range, which is power of 2, shift right to extract each bit. Although consequentially shift or mul/div is equal and compile should able to optimize to most fast one, however keep it shift left/right makes the entire logic more clearly.